pkg/local_workflows/whoami_workflow_test.go

```diff
@@ -249,7 +249,7 @@ func Test_WhoAmI_whoAmIWorkflowEntryPoint_extractUserFailures(t *testing.T) {
 	networkAccessMock.EXPECT().GetHttpClient().Return(mockClient).AnyTimes()
 
 	// setup
-	expectedError := errors.New("error fetching user data: error while extracting user: missing property 'username'")
+	expectedError := errors.New("error fetching user data: error while extracting user: missing properties username/name/email")
 
```

Comment:
**issue (blocking):** Introduce a test that verifies we correctly handle the Service account response payload from the /self endpoint. This was the issue that is currently blocking upgrading the go-application-framework.